hack/clean.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# **clean.sh**
+
+cd $GOPATH/src/github.com/kubeedge/kubeedge
+
+# if the edge_core file is not present then  count is 0 for the first time
+if [ ! -f /tmp/edge_core.log1.tar.gz ]; then
+count=0
+else
+count=$(ls /tmp/edge_core.log*.tar.gz | wc -l)
+fi
+
+remove_folder() {
+if [ -d bin ]; then
+rm -rf bin
+fi
+}
+
+#terminate mosquitto if running
+stop_mosquitto() {
+if  pgrep mosquitto  >/dev/null 2>&1  ; then
+     pkill -9 mosquitto
+fi
+}
+
+#terminate edge core if running
+stop_edge_core () {
+if pgrep edge_core >/dev/null 2>&1 ; then
+     pkill -9 edge_core
+fi
+}
+
+#delete logs,covergage and database related files
+cleanup_files () {
+if [ -f edge_core ]; then
+rm -f edge_core
+fi
+
+if [[ -f edge_core.log  &&  $count -eq 0 ]] ; then #if edgecore_log.tar file not exists and count is zero then its tar for the first time
```

Comment:
I feel we should handle this using logrotate utility. let's not introduce log rotation in this cleanup activity. it better we have a separate issue logged for log rotation.

Reply:
ok